mlrun/common/constants.py

```diff
@@ -14,7 +14,7 @@
 #
 
 IMAGE_NAME_ENRICH_REGISTRY_PREFIX = "."  # prefix for image name to enrich with registry
-MLRUN_MODEL_CONF = "model-conf"
+MLRUN_MODEL_CONF = "serving-conf"
```

Comment:
If model-conf is misleading I would change the MLRUN_MODEL_CONF as well

Reply:
Right, missed that...